### PR TITLE
Wall 3858

### PR DESCRIPTION
**Pull request by mohammedluqmaan-deriv**
Added switch all available wallets.

### CODE REVIEW

**Review by afsaneh-rezaeinejad-deriv — APPROVED** [review]
LGTM

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
I'm not seeing any testing library queries here for the locators.
This may often be the case with Wallets but could you double-check?

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]
we have existing test covered ETH and BTC, can you please update the remaining in the existing test?

**Comment by mohammedluqmaan-deriv** [comment]
> we have existing test covered ETH and BTC, can you please update the remaining in the existing test?



> we have existing test covered ETH and BTC, can you please update the remaining in the existing test?

Added the test in existing file cypress/e2e/smoke/Wallets/walletSwitcher.cy.js, Thank you.

**Comment by mohammedluqmaan-deriv** [comment]
> I'm not seeing any testing library queries here for the locators. This may often be the case with Wallets but could you double-check?

I did use few things like aliases and common looping for [actions] but will get to know about testing library queries as-well, Thanks

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]
we have existing test QATEST-139905 in this file:
…/cypress/e2e/smoke/Wallets/wallerCashierRedirection.cy.js

Can you please help to update in this file instead?

**Review by afsaneh-rezaeinejad-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/smoke/Wallets/walletCashierRedirection.cy.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+describe('QATEST-139905 - Mobile wallet card redirection', () => {
+  beforeEach(() => {
+    cy.c_login({ user: 'walletloginEmail' })
+  })
+  it('should be able to switch and open deposit, withdraw and transfer in all the available wallets in Responsive', () => {
+    cy.c_visitResponsive('/', 'small')
+    cy.c_skipPasskeysV2()
```
[the remark this answers is not in the document]
Comment (mohammedluqmaan-deriv) [reply cypress/e2e/smoke/Wallets/walletCashierRedirection.cy.js]: Thank you have removed it.

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**